lib/cmd/fh3/resources/list.js

```diff
@@ -0,0 +1,21 @@
+/* globals i18n */
+var fhc = require("../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('List Resources across RHMAP'),
```

Comment:
Maybe change this to `List resource usage such as CPU and Memory for a given environment`?